uniform/lib/UniForm.php

```diff
@@ -362,6 +362,23 @@ public function hasError($key = false)
 	}
 
 	/**
+	* Checks if a field is a required field or not.
+	*
+	* Returns `true` if the field was in the list of required fields.
+	* Returns `false` otherwise.
+	*
+	* @param string $key the key of the form field to check.
+	*
+	* @return boolean
+	*/
+	public function isRequired($key)
+	{
+		return 	!is_null($key) &&
+			!is_null($this->options['required']) &&
```

Comment:
I'm sorry that I'm being so lazy reviewing your code but here I found another issue. I'd rather use `array_key_exists` for checking the arrays, otherwise you'd get a notice `Undefined index`. So it'd be:
```php
array_key_exists('required', $this->options) && array_key_exists($key, $this->options['required'])
```